types/types.cc

```diff
@@ -89,18 +89,15 @@ time_point_to_string(const T& tp)
     };
 
     auto millis = d.rem;
-    if (!millis) {
-        return fmt::format("{}", to_string(tm));
-    }
     // adjust seconds for time points earlier than posix epoch
     // to keep the fractional millis positive
     if (millis < 0) {
         millis += 1000;
         seconds--;
         gmtime_r(&seconds, &tm);
     }
-    auto micros = millis * 1000;
-    return fmt::format("{}.{:06d}", to_string(tm), micros);
+    millis = std::round(millis * 1000) / 1000;
+    return fmt::format("{}.{:03d}Z", to_string(tm), millis);
```

Comment:
this changes the output like: 
```diff
- "2011-02-03T04:05:00"
+ "2011-02-03T04:05:00.000Z"
```

so if we want to go in this direction. we might want to update the tests in https://github.com/scylladb/scylla-dtest/blob/next/json_test.py#L50 accordingly

Reply:
sorry, 404

Reply:
@tchaikov scylla-dtest.git is private repository, it is not available to external contributors (not sure why).

Reply:
sorry, i just realized this.

Reply:
> @tchaikov scylla-dtest.git is private repository, it is not available to external contributors (not sure why).

It was like that from day one, I also don't know why. 
@alezzqz you don't need to worry too much, the better (in my opinion) functional test frameworks are all in scylla.git itself and all open-source.